static/js/tools/map/util.ts

```diff
@@ -572,47 +573,55 @@ function getMetahashMap(
  * @param placeStatDateWithinPlace
```

Comment:
update the function comment? Also, include that observationDates is arranged in order (if that's correct)?